dev/packages/example/cisco-1.0.0/dataset/asa/filebeat/module.yml

```diff
@@ -0,0 +1,3 @@
+- module: cisco
```

Comment:
I don't think moving forward we need all the filebeat files?

Reply:
I was keeping them here because the coredns package had them. I wasn't sure where they were needed  and I could not test end-to-end because fleet was not working in ingest manager.